flexmeasures/data/scripts/clean_database.sh

```diff
@@ -60,9 +60,11 @@ function create_database() {
           grant_privileges $1 $2
       else
         # if user doesn't exist, first create it and then give the permissions.
-        if create_user $2
+        if ! create_user $2
           then
-            grant_privileges $1 $2
+            exit 1
```

Comment:
Let's be consistent. All functions should return, not exit.

No need to change the main loop, as right now nothing is called afterwards, just make this function consistent, then we can merge.